docs/kubernetes-reliability.md

```diff
@@ -0,0 +1,101 @@
+Distributed system such as Kubernetes are designed to be resilient to the
```

Comment:
Done. I was thinking but was not able to fine a good phrase. I like 'overview' very much. Thanks for tip